userspace/falco/app/options.cpp

```diff
@@ -19,6 +19,9 @@ limitations under the License.
 #include "../configuration.h"
 #include "config_falco.h"
 
+// disable cxxopts vector delimiter, meaning that
+// -o test1,test2,test3 won't be treated like -o test1 -o test2 -o test3
+#define CXXOPTS_VECTOR_DELIMITER '\0'
```

Comment:
This may be seen as a (minor) breaking change in the CLI context. Is there any other available alternative?

Reply:
It does not look like it as it happens unconditionally for `vector<T>` values: https://github.com/jarro2783/cxxopts/blob/63d1b65a694cfceafc20863afa75df49dfbe6b2a/include/cxxopts.hpp#L1100